pkg/daemon/command/command.go

```diff
@@ -122,12 +124,23 @@ func WithThreadNum(num int) Opt {
 	}
 }
 
-func WithConfig(config string) Opt {
+func WithConfig(id string) Opt {
 	return func(cmd *DaemonCommand) {
+		config := strings.Replace("/api/v1/daemons/{id}/config", "{id}", id, 1)
 		cmd.Config = config
 	}
 }
 
+func WithConfigSource(source string) Opt {
+	return func(cmd *DaemonCommand) {
+		if source == "" {
+			cmd.ConfigSource = "/run/containerd-nydus/system.sock"
```

Comment:
It would be better to define "/run/containerd-nydus/system.sock" as a constant.

Reply:
I need to find a method to obtain the Controller instance to obtain this value.